rpcperms/interceptor.go

```diff
@@ -120,6 +188,14 @@ func (r *RPCInterceptor) CreateServerOpts() []grpc.ServerOption {
 		strmInterceptors, errorLogStreamServerInterceptor(r.rpcsLog),
 	)
 
+	// Add RPC state check interceptors.
+	unaryInterceptors = append(
+		unaryInterceptors, r.rpcStateUnaryServerInterceptor(),
```

Comment:
Should the state interceptors come before the macaroon interceptors?

Reply:
I think you are right, that makes more sense! Done.

I also moved the logging interceptors first, makes sense that all calls are logged, whether they pass authentication or not.